snippets/vectorAngle.md

```diff
@@ -0,0 +1,28 @@
+---
+title: vectorAngle
+tags: math, beginner
+---
+
+Returns the angle (theta) value between two vectors.
+
+Use `Math.trunc()` to convert the half of `length` to integer. 
+Use `Array.prototype.slice()` slice the incoming array into two vectors. 
+Use `Array.prototype.reduce()`, `Math.pow()` and `Math.sqrt()` to calculate the magnitude of each vector and the scalar product of the two vectors.
+Use `Math.acos()` to calculate arccos and get the theta.
+
+
+```js
+const vectorAngle = (...x) =>{ 
```

Comment:
I am not 100% sure that using a vararg here is the right call. We could use `x`, and `y`, which would be two arrays, what do you think?